packages/ui/src/snackbar/SnackbarContent.tsx

```diff
@@ -93,13 +102,21 @@ export const SnackbarContent: ForwardRefExoticComponent<SnackbarContentProps> =
       },
       ref,
     ) => {
-      const { icon, closeButton, variantError, variantSuccess, ...styles } =
-        useStyles({ classes });
+      const {
+        icon,
+        closeButton,
+        variantError,
+        variantSuccess,
+        variantErrorOutline,
+        ...styles
+      } = useStyles({ classes });
       const Icon =
         variant === 'error'
           ? Warning
           : variant === 'success'
           ? CheckCircle
+          : variant === 'error-outline'
+          ? Error
```

Comment:
@TravnikovDev what about `success` and `default` type?
Can we get `current-color` instead of specific one and use it for outline?

Reply:
what about add one more props `outline={True | False}` and get currentColor